lib/remote_retro/emails.ex

```diff
@@ -0,0 +1,31 @@
+defmodule RemoteRetro.Emails do
+  import Bamboo.Email, except: [from: 2]
+  import Ecto.Query
+  alias RemoteRetro.Repo
+  def action_items_email(email_address, retro_id) do
+    new_email(
```

Comment:
todo: retrieve the action items from the database here, so that we don't hit the database once for the html email and once for the text email